From 54fcbaa55006237eccf703e514a245b9650a7bca Mon Sep 17 00:00:00 2001
From: Eddie Huang <eddie.huang@mediatek.com>
Date: Fri, 30 Jan 2015 18:21:39 +0800
Subject: [PATCH 10/10] Fix: soc: mediatek: PMIC pwrapper

Fix pwrapper error that base address name not align with device tree.

Change-Id: I4e64f3b3a5275748808fe8a77141598b86090861
Signed-off-by: Eddie Huang <eddie.huang@mediatek.com>
---
 drivers/soc/mediatek/mtk-pmic-wrap.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/soc/mediatek/mtk-pmic-wrap.c b/drivers/soc/mediatek/mtk-pmic-wrap.c
index 4989b0f..c91a046 100644
--- a/drivers/soc/mediatek/mtk-pmic-wrap.c
+++ b/drivers/soc/mediatek/mtk-pmic-wrap.c
@@ -914,7 +914,7 @@ static int pwrap_probe(struct platform_device *pdev)
 	wrp->arb_en_all = type->arb_en_all;
 	wrp->dev = &pdev->dev;
 
-	res = platform_get_resource_byname(pdev, IORESOURCE_MEM, "pwrap");
+	res = platform_get_resource_byname(pdev, IORESOURCE_MEM, "pwrap-base");
 	wrp->base = devm_ioremap_resource(wrp->dev, res);
 	if (IS_ERR(wrp->base))
 		return PTR_ERR(wrp->base);
@@ -928,7 +928,7 @@ static int pwrap_probe(struct platform_device *pdev)
 
 	if (pwrap_is_mt8135(wrp)) {
 		res = platform_get_resource_byname(pdev, IORESOURCE_MEM,
-				"pwrap-bridge");
+				"pwrap-bridge-base");
 		wrp->bridge_base = devm_ioremap_resource(wrp->dev, res);
 		if (IS_ERR(wrp->bridge_base))
 			return PTR_ERR(wrp->bridge_base);
-- 
1.9.1

